### DISCUSSION TITLE
I can't play afreecatv video using API

### DISCUSSION THREAD

**Discussion by chanuuuuu** [streamlink/streamlink#3649] [Q&A]
I try to access the stream of afreeca tv using API. I used the code below.

```
session = streamlink.Streamlink()
streams = session.streams('http://play.afreecatv.com/rrvv17/231925269')
print(streams['sd'])
```
A stream was extracted using url from a certain creator of afreecatv.

```<HLSStream('http://pc-web.stream.afreecatv.com/live-stm-17/auth_playlist.m3u8?aid=.A32.7bbT56vyHM9fKZk.ron0fnIGf43OVz5qjn75oBzLaNN10VMEhfVoDChVkDiumpPzvKoWlxj43uwfjm0OWzeKrEPeDX-uxVaPkhqF31Dq7txn8wKpsOj5gnVvs_U', None)>```

This stream url doesn't play.

**Comment by back-to** [comment]
you will need to use headers, cookies and so on.

you can get it with `streams['sd'].__json__()`
